go.mod

```
@@ -1,20 +1,28 @@
 module github.com/blalor/flight-cal
 
+go 1.23.0
```

```suggestion
go 1.23
```

this fixes my error that i had 
```
$ make
go: errors parsing go.mod:
/home/atticus/github/flight-cal/go.mod:3: invalid go version '1.23.0': must match format 1.23
go: errors parsing go.mod:
/home/atticus/github/flight-cal/go.mod:3: invalid go version '1.23.0': must match format 1.23
go build -o .godeps/bin/ginkgo github.com/onsi/ginkgo/ginkgo
go: errors parsing go.mod:
/home/atticus/github/flight-cal/go.mod:3: invalid go version '1.23.0': must match format 1.23
make: *** [Makefile:27: .godeps/bin/ginkgo] Error 1
```